go/vt/dbconnpool/pooled_connection.go

```diff
@@ -28,6 +29,10 @@ type PooledDBConnection struct {
 	pool *ConnectionPool
 }
 
+func (pc *PooledDBConnection) Expired(lifetimeTimeout time.Duration) bool {
+	return false
+}
+
 func (pc *PooledDBConnection) ApplySetting(context.Context, *pools.Setting) error {
```

Comment:
PooledDBConnection would also need same implementation as `dbconn`

Reply:
https://github.com/vitessio/vitess/pull/11337#discussion_r984521887
Does `PooledDBConnection` also need expiration based on `maxLifetime`?

Reply:
Sorry for the confusion, `PooledDBConnection` would also need expiration logic.